doc/admin-guide/files/records.config.en.rst

```diff
@@ -1354,28 +1354,84 @@ Origin Server Connect Attempts
    way up to ts:cv:`proxy.config.net.connections_throttle` connections, which
    in turn can starve incoming requests from available connections.
 
-.. ts:cv:: CONFIG proxy.config.http.origin_max_connections INT 0
+.. ts:cv:: CONFIG proxy.config.http.per_server.max_connections INT 0
    :reloadable:
    :overridable:
 
-   Limits the number of socket connections per origin server to the value specified. To enable, set to one (``1``).
+   Set a limit for the number of concurrent connections to an upstream server group. A value of
+   ``0`` disables checking. If a transaction attempts to connect to a group which already has the
+   maximum number of concurrent connections the transaction either rechecks after a delay or a 503
+   (``HTTP_STATUS_SERVICE_UNAVAILABLE``) error response is sent to the user agent. To configure
 
-.. ts:cv:: CONFIG proxy.config.http.origin_max_connections_queue INT -1
+   Number of transactions that can be delayed concurrently
+      See :ts:cv:`proxy.config.http.per_server.queue_size`.
+
+   How long to delay before rechecking
+      See :ts:cv:`proxy.config.http.per_server.queue_delay`.
+
+   Upstream server group definition
+      See :ts:cv:`proxy.config.http.per_server.match`.
+
+   Frequency of alerts
+      See :ts:cv:`proxy.config.http.per_server.alert_delay`.
+
+.. ts:cv:: CONFIG proxy.config.http.per_server.match STRING ip
    :reloadable:
    :overridable:
 
-   Limits the number of requests to be queued when the :ts:cv:`proxy.config.http.origin_max_connections` is reached.
-   When disabled (``-1``) requests are will wait indefinitely for an available connection. When set to ``0`` all
-   requests past the :ts:cv:`proxy.config.http.origin_max_connections` will immediately fail. When set to ``>0``
-   ATS will queue that many requests to go to the origin, any additional requests past the limit will immediately fail.
+   Control the definition of an upstream server group for
+   :ts:cv:`proxy.config.per_server.max_connections`. This must be one of the following keywords.
+
+   ip
+      Group by IP address. Each IP address is a group.
+
+   port
+      Group by IP address and port. Each distinct IP address and port pair is a group.
+
+   host
+      Group by host name. The host name is the post remap FQDN used to resolve the upstream
+      address.
+
+   both
+      Group by IP address, port, and host name. Each distinct combination is a group.
```

Comment:
So a host name can sometimes resolve to multiple IP addresses, presumably for some load-balancing purpose?  Otherwise, you'd only want to look at the FQDN / port combination.

Reply:
Yes, a single FQDN can resolve to multiple IP addresses.